ts/RootContainer.tsx

```diff
@@ -41,6 +41,10 @@ class RootContainer extends React.PureComponent<Props> {
   }
 
   private handleBackButton = () => {
+    if (this.props.navigation.index === 0) {
+      BackHandler.exitApp();
+      return false;
```

Comment:
I am quite sure that if you handle the back button press you have to return true.